visualization/app/codeCharta/core/statistic/statistic.service.ts

```diff
@@ -42,23 +42,23 @@ export class StatisticMapService {
         }
 
         let accumulated = {} as CodeMap;//Map that contains an array of every value of every map given in maps array
-        accumulated.root = {} as CodeMapNode;
+        accumulated.nodes = {} as CodeMapNode;
         let unified: CodeMap;
         accumulated.fileName = operation+"";
         for(let i: number=0; i<maps.length; i++){//Loop through every CodeMap of the input array and get all of them in accumulated
```

Comment:
boyscout: remove comments?

Reply:
The `statistic.service` gets deleted in PR #329